### PR TITLE
Us23 n2 yo sattelites 3 d map #41

### PR DESCRIPTION
**Pull request by gnos28**

### CODE REVIEW

**Review by PaulKip44 — APPROVED** [review]
ok

**Comment by gnos28** [comment]
https://github.com/ericgio/react-bootstrap-typeahead/issues/715
please notice I got this warning reported 2 days ago on react-bootstrap-typeahead package github
let's keep an eye over this, hopefully it'll be quickly solved

**Comment by gnos28** [comment]
image
some improvements can be done on initial loading time
- useless packages loaded (why react-calendar ??)
- 2mb coming from backend >> database should be cleaned up a bit
- earth texture coming from external source >> should be put in local
a loading component will also be added in a near future

**Comment by gnos28** [comment]
image
known performance issue / bug :
when hovering a satellite (in order to display details in the bottom part) seems like the whole component / page ? is being re-rendered from scratch
to be investigated though...

File: backend/src/models/N2yoManager.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  // | satlng         | double(12,8) | YES  |     | NULL    |                |
+  // | satalt         | double(12,4) | YES  |     | NULL    |                |
+
+  // update(item) {
+  //   return this.connection.query(
+  //     `update ${ItemManager.table} set title = ? where id = ?`,
+  //     [item.title, item.id]
+  //   );
+  // }
+}
```
Comment (lchastanet) [inline backend/src/models/N2yoManager.js:36]: Dead code 😄

File: backend/src/N2yoListener.js [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+  },
+  {
+    id: 53,
+    name: "OneWeb",
+  },
+  {
+    id: 54,
+    name: "Chinese Space Station",
+  },
+];
```
Comment (lchastanet) [inline backend/src/N2yoListener.js:218]: Please put data in dedicated file

**Review by lchastanet — CHANGES_REQUESTED** [review]

File: backend/src/models/N2yoManager.js (hunk 1 above)
Comment (gnos28) [reply backend/src/models/N2yoManager.js:36]: already corrected in last commit

File: backend/src/N2yoListener.js (hunk 2 above)
Comment (gnos28) [reply backend/src/N2yoListener.js]: done in last commit

**Review by lchastanet — APPROVED** [review]
Ok, nickel

**Pull request merged**